modules/jolt_physics/objects/jolt_body_3d.h

```diff
@@ -287,10 +287,16 @@ class JoltBody3D final : public JoltShapedObject3D {
 	void set_collision_priority(float p_priority) { collision_priority = p_priority; }
 
 	DampMode get_linear_damp_mode() const { return linear_damp_mode; }
-	void set_linear_damp_mode(DampMode p_mode) { linear_damp_mode = p_mode; }
+	void set_linear_damp_mode(DampMode p_mode) {
+		linear_damp_mode = p_mode;
+		_update_damp();
+	}
```

Comment:
Move this implementation to `jolt_body_3d.cpp` instead, ideally while preserving the same order as the declarations in the header file, so beneath `JoltBody3D::set_angular_damp`.

I would suggest adding a check to see whether the value has changed as well, since this could otherwise prevent a body from sleeping if continously set to the same value.

```suggestion
	void set_linear_damp_mode(DampMode p_mode) {
		if (p_mode == linear_damp_mode) {
			return;
		}

		linear_damp_mode = p_mode;
		_update_damp();
	}
```